### PR TITLE
Make `markdownlint-cli2` more discoverable

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ To run `markdownlint-cli` as part of a [pre-commit][pre-commit] workflow, add so
 ## Related
 
 - [markdownlint][markdownlint] - API for this module
+- [markdownlint-cli2][markdownlint-cli2] - another CLI
```
Comment (DavidAnson) [inline README.md:147]: Thanks! Could you please capitalize'A' for consistency?

File: README.md [hunk 2]
```diff
@@ -161,6 +162,7 @@ MIT © Igor Shubovych
 [config]: https://github.com/DavidAnson/markdownlint#optionsconfig
 [style-folder]: https://github.com/DavidAnson/markdownlint/tree/main/style
 [rc-standards]: https://www.npmjs.com/package/rc#standards
+[markdownlint-cli2]: https://github.com/DavidAnson/markdownlint-cli2
```
Comment (DavidAnson) [inline README.md:165]: This list is not fully sorted, but could you move this line below "markdownlint", please?

**Review by DavidAnson — CHANGES_REQUESTED** [review]
Good idea!

**Review by DavidAnson — APPROVED** [review]

**Comment by AlekSi** [comment]
Oops, sorry, missed your comments. Thank you!

**Pull request merged**